ExpressWash/ExpressWash/Car+Convenience.swift

```diff
@@ -16,47 +16,55 @@ enum CarSize: String {
 }
 
 extension Car {
-    convenience init(carId: Int32 = 0,
+    convenience init(clientId: Int32,
+                     carId: Int32 = 0,
                      make: String,
                      model: String,
                      year: Int16,
                      color: String,
                      licensePlate: String,
-                     photo: URL?,
+                     photo: String?,
+                     category: String,
                      size: String,
                      context: NSManagedObjectContext = CoreDataStack.shared.mainContext) {
         self.init(context: context)
+        self.clientId = clientId
         self.carId = carId
         self.make = make
         self.model = model
         self.year = year
         self.color = color
         self.licensePlate = licensePlate
         self.photo = photo
+        self.category = category
         self.size = size
     }
 
     convenience init(representation: CarRepresentation,
                      context: NSManagedObjectContext = CoreDataStack.shared.mainContext) {
         self.init(context: context)
-        self.carId = Int32(representation.carId)
+        self.clientId = Int32(representation.clientId)
+        self.carId = Int32(representation.carId ?? NOID)
         self.make = representation.make
         self.model = representation.model
         self.year = representation.year
         self.color = representation.color
         self.licensePlate = representation.licensePlate
         self.photo = representation.photo
+        self.category = representation.category
         self.size = representation.size
     }
 
     var representation: CarRepresentation? {
-        CarRepresentation(carId: Int(self.carId),
+        CarRepresentation(clientId: Int(self.owner!.userId),
```

Comment:
See here you are accessing the `owner!.userId` in the way I described above. I think adding the Car.clientId property is superfluous and may lead to confusion/problems later.